### PR TITLE
Updated contributing documentation

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Updated contributing documentation.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,14 +24,23 @@ cd yamada-colors
...
 4. Run `pnpm dev` to start the local server.
 
-## Commands
+## Development
+
+To improve our development process, we have set up tools and systems.
+
+### Tooling
+
+- [Next.js](https://nextjs.org/) Framework for web sites.
```
Comment (taroj1205) [inline CONTRIBUTING.md:33]: Framework for the websiteだと思います。

Comment (hirotomoyamada) [reply CONTRIBUTING.md:33]: Fixed

Comment (taroj1205) [reply CONTRIBUTING.md]: for "the" websiteに直してください。

Comment (hirotomoyamada) [reply CONTRIBUTING.md]: Fixed

**Review by taroj1205 — APPROVED** [review]
Looks good to me!

**Pull request merged**